cds_dojson/marc21/fields/utils.py

```diff
@@ -309,39 +309,79 @@ def _get_correct_video_contributor_role(role):
     return tranlations[role.lower()]
 
 
-def _extract_json_ids(info):
+def _get_correct_books_contributor_role(role):
+    """Clean up roles."""
+    tranlations = {
```

Comment:
very very minor, typo: tranlations -> translations

Reply:
well spotted, thanks! Fixed in all 4 places.